skimage/exposure/tests/test_exposure.py

```diff
@@ -748,6 +748,14 @@ def test_is_low_contrast():
     assert not exposure.is_low_contrast(image, upper_percentile=100)
 
 
+def test_is_low_contrast_boolean():
+    image = np.zeros((8, 8))
```

Comment:
```suggestion
    image = np.zeros((8, 8), dtype=bool)
```

Reply:
Definitely need `bool` here! This actually uncovered a copy paste error in the new code above!